jstreegrid.js

```diff
@@ -41,11 +41,13 @@
     getIds = function (nodes) {
       return $.makeArray(nodes.map(function(){return this.id;}));
     },
-		findDataCell = function (uniq, ids, col) {
-			var ret = $(), columns = [].concat(col), cellId;
+    findDataCell = function (uniq, ids, col, scope) {
```

Comment:
What is the point of `scope` here? In normal language, what does `findDataCell()` do differently? Before, its purpose was: given: a tree (`uniq`); 0, 1 or more `id` of nodes,; 1 or more columns `col`; then find the actual cells that contain that data. Basically, it is like finding a series of cells on an Excel spreadsheet given the rows (`id`) and columns (`col`). 

What does it do now with `scope`?